keyboards/ut472/ut472.c

```diff
@@ -0,0 +1,15 @@
+#include "ut472.h"
+#include "led.h"
```

Comment:
I don't actually see this file being added.  

If it doesn't exist, it should be removed.

Reply:
go ahead and remove them or I can

Reply:
Well, to make it easier for you: 
```suggestion
```